### DISCUSSION TITLE
Why doesn't the editor convert the title to a paragraph after the node is split?

### DISCUSSION THREAD

**Discussion by AxelBriche** [ueberdosis/tiptap#2652] [General Questions] [answered]
Why the editor doesn't convert the node type here?

`content: 'heading block*'`
That config allow only one "heading", followed by zero or more blocks.

![CleanShot 2022-03-27 at 17 06 53](https://user-images.githubusercontent.com/1295251/160287942-6be61237-10f7-4566-b933-e9ad8cc350ec.gif)
([Try it here](https://tiptap.dev/examples/custom-document))

Same problem with blockquote and lists, the editor can convert a blockquote to a list, but not a list to a blockquote:

![CleanShot 2022-03-27 at 17 10 46](https://user-images.githubusercontent.com/1295251/160288108-0fc2fbfb-f770-415c-8044-77960316ebac.gif)

Why does the editor not perform a node conversion in this situation [like Slate in this example](https://www.slatejs.org/examples/forced-layout)? 

![CleanShot 2022-03-27 at 18 16 30](https://user-images.githubusercontent.com/1295251/160290733-68e39037-e026-44f1-93ae-a5a3fc32f548.gif)

**Comment by philippkuehn** [comment]
Oh you are right. I’m pretty sure this has worked before. I’ll have a look.

**Reply by AxelBriche** [reply]
This is because heading is inside the "block" group.
I have fixed the problem with a custom extension, a different group and a custom document:

```
const CustomDocument = Document.extend({
  content: 'title block*',
})
```

and...

<img width="471" alt="CleanShot 2022-03-30 at 20 02 20@2x" src="https://user-images.githubusercontent.com/1295251/160901454-08f4b487-6cc4-4a20-aed3-98d88e85b01c.png">

--------

Another subject :
I want create a "spoiler" extension, how can I share the code with the community please?
Same question for the translation of the documentation into French, how can I help?

**Comment by AxelBriche** [comment, marked as answer]
Solved here: https://github.com/ueberdosis/tiptap/discussions/2652#discussioncomment-2472097
